mlrun/feature_store/common.py

```diff
@@ -81,6 +81,19 @@ def get_feature_set_by_uri(uri, project=None):
     """get feature set object from db by uri"""
     db = mlrun.get_run_db()
     project, name, tag, uid = parse_feature_set_uri(uri, project)
+
+    fs_project_name = [project, name]
+    mlrun.api.utils.clients.opa.Client().query_project_resources_permissions(
+        mlrun.api.schemas.AuthorizationResourceTypes.feature_set,
+        project,
+        lambda feature_set_project_name_tuple: (
+            fs_project_name[0],
+            fs_project_name[1],
+        ),
+        mlrun.api.schemas.AuthorizationAction.read,
+        mlrun.api.schemas.AuthInfo(),
+    )
```

Comment:
You're in client context here, you can't use opa client, you should use db.verify_authorization()